lib/hostDrawer.js

```diff
@@ -110,7 +137,13 @@ module.exports = (api) => {
 					h('div', `Participant Count: ${participantState.length}`),
 					h('div', participantState.map(participant => {
 						return h('div',`Participant Name: ${participant.displayName}`)
-					}))
+					})),
+					h('hr',),
+					h('div.Drawer_Title', 'external api sample'),
+					h('p','this is an example for using webhooks to make an external api call'),
+					h('button.Button', { 'ev-click': ()=>triggerWebhook() }, 'Perform webhook api call' ),
```

Comment:
```
() => triggerWebhook()
```

Reply:
```suggestion
					h('button.Button', { 'ev-click': triggerWebhook }, 'Perform webhook api call' ),
```

P.S. why not setup husky + lint-staged + prettier (or similar)?

Reply:
Yes, Karim is working on it